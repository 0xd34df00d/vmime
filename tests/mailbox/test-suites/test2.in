mailbox1 <mailbox@one>,;,,,	,,  	,,;group1:mailbox1@group1,  mailbox2@group2,,"mailbox #3" <mailbox3@group2>;, <mailbox@two>,,,,,,,,=?iso-8859-1?q?mailbox_number_3?= <mailbox@three>, =?abc?Q?mailbox?=   	 =?def?Q?_number_4?= <mailbox@four>
