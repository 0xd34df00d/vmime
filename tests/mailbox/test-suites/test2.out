MAILBOX[name=[[us-ascii,mailbox1]],email=mailbox@one]
GROUP[name=[[us-ascii,group1]]]
* MAILBOX[name=[],email=mailbox1@group1]
* MAILBOX[name=[],email=mailbox2@group2]
* MAILBOX[name=[[us-ascii,mailbox #3]],email=mailbox3@group2]
MAILBOX[name=[],email=mailbox@two]
MAILBOX[name=[[iso-8859-1,mailbox number 3]],email=mailbox@three]
MAILBOX[name=[[abc,mailbox],[def, number 4]],email=mailbox@four]
